ci/run_slither.sh

```diff
@@ -9,8 +9,8 @@ run_slither() {
     truffle compile 
 
     cd $PROTOCOL_DIR
-    slither --exclude=naming-convention,solc-version,pragma,external-function,reentrancy-benign,reentrancy-no-eth,arbitrary-send,locked-ether,reentrancy-eth $1
+    slither --exclude=naming-convention,solc-version,pragma,external-function,reentrancy-benign,reentrancy-no-eth,arbitrary-send,locked-ether,reentrancy-eth,uninitialized-state-variables $1
```

Comment:
What uninitialized variable was it complaining about?

Reply:
`_accountBalanceSnapshots` inside of ERC20Snapshot.sol (not in our code).

Reply:
I see. That's annoying.